WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -805,8 +805,8 @@ class AbstractPostListViewController: UIViewController, WPContentSyncHelperDeleg
         WPAnalytics.track(.postListPublishAction, withProperties: propertiesForAnalytics())
 
         apost.status = PostStatusPublish
-        if let date = apost.dateCreated(), date == (Date() as NSDate).laterDate(date) {
-            apost.setDateCreated(Date())
+        if let date = apost.dateCreated, date == (Date() as NSDate).laterDate(date) {
```

Comment:
@SergioEstevao @diegoreymendez in this PR i've converted the `dateCreated()` method into an actual property. Please, note that the code / logic wasn't changed, just the way the data is accessed.

Thanks for the heads though!